modules/textual_inversion/textual_inversion.py

```diff
@@ -295,9 +384,12 @@ def train_embedding(embedding_name, learn_rate, batch_size, data_root, log_direc
 
             p = processing.StableDiffusionProcessingTxt2Img(
                 sd_model=shared.sd_model,
+                prompt=preview_prompt,
                 do_not_save_grid=True,
                 do_not_save_samples=True,
                 do_not_reload_embeddings=True,
+                negative_prompt=preview_prompt.replace(ds.placeholder_token, ds.placeholder_token + '-uc') if use_negative else None,
+                cfg_scale=cfg_scale if use_negative else 1.0,

```

Comment:
Are you sure setting CFG to 1.0 when not using negative? 
By default I think this is set to 7.0 when initializing the parent class:
https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/836c8ea9b7980b5290cca7d67cc922d9edb8cc42/modules/processing.py#L56